ledger-core/virtual/execute/execute.go

```diff
@@ -518,9 +517,19 @@ func (s *SMExecute) stepExecuteDecideNextStep(ctx smachine.ExecutionContext) sma
 		// send VCallResult here
 		return ctx.Jump(s.stepSaveNewObject)
 	case execution.Error:
-		err := throw.W(newState.Error, "failed to execute request")
-		ctx.Log().Warn(err)
-		s.prepareExecutionError(err)
+		var detail runner.ErrorDetail
+		if throw.FindDetail(newState.Error, &detail) {
```

Comment:
just in case - it can also be a one-liner

if d := runner.ErrorDetail{}; throw.FindDetail(newState.Error, &d) {
...
}

Reply:
Thanks, fixed. But had to use `new(runner.ErrorDetail)`